dark/bowtie2.py

```diff
@@ -166,10 +166,11 @@ def removePrimers(self, bedFile):
             raise ValueError('makeBAM() has not yet been called.')
 
         self._report("removing primers specified in %s" % bedFile)
+        tempTrimmedBam = "%s.trimmed" % self._bamFile
         self._executor.execute(
-            "ivar trim -b %s -p result-trimmed -i %s -q 20 -m 30 -s 4 -e" %
-            (bedFile, self._bamFile))
-        self._executor.execute("mv result-trimmed.bam '%s'" % self._bamFile)
+            "ivar trim -b %s -p %s -i %s -q 20 -m 30 -s 4 -e" %
```

Comment:
Can you put these `%s` args into single quotes, just to eliminate the possibility that they might have spaces or shell metacharacters.